Libraries/Boards/MAX32570/MN_EvKit_V1/Include/board.h

```diff
@@ -38,12 +38,16 @@
  * @brief   Board support package API.
  */
 
-#include <stdio.h>
-#include "spi.h"
-
 #ifndef LIBRARIES_BOARDS_MAX32570_MN_EVKIT_V1_INCLUDE_BOARD_H_
 #define LIBRARIES_BOARDS_MAX32570_MN_EVKIT_V1_INCLUDE_BOARD_H_
 
+#include <stdio.h>
+#include "spi.h"
+#define LED_OFF 1 /// Override inactive state of LEDs
+#define LED_ON 0 /// Override active state of LEDs
```

Comment:
I know this line added here to define LED_ON/OFF before it does in led.h file.
But Instead of #defining between includes lines i recommend using #undef  command

In that way you can keep existing line as below.

#ifdef LED_OFF
#undef LED_OFF
#endif   
#define LED_OFF  X

Same for LED_ON

additionally led.h file shall never includes in any example. Need to be checked.

Reply:
https://github.com/Analog-Devices-MSDK/msdk/pull/613/commits/af7cbafb3edf54de1a9e6de86961bfb8c19e9d55